tests/test_kerberos_client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from src.kerberos_client import KerberosClient
+from shared_server import *
+
+
+@pytest.fixture
+def client():
+    return KerberosClient()
```

Comment:
Also add the tests:
* accept new client (Assert name and pass)
* use a me.info file and see if you can register
* send a key to server